node/state_api/src/lib.rs

```diff
@@ -0,0 +1,397 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use actor::{
+    miner::{
+        compute_proving_period_deadline, ChainSectorInfo, DeadlineInfo, Deadlines, Fault,
+        MinerInfo, SectorOnChainInfo, SectorPreCommitOnChainInfo, State,
+    },
+    power::Claim,
+};
+use address::Address;
+use async_std::sync::Arc;
+use async_std::task;
+use bitfield::BitField;
+use blocks::{Tipset, TipsetKeys};
+use blockstore::BlockStore;
+use chain::ChainStore;
+use cid::Cid;
+use clock::ChainEpoch;
+use fil_types::SectorNumber;
+use message::{MessageReceipt, UnsignedMessage};
+use num_bigint::BigUint;
+use num_traits::identities::Zero;
+use state_manager::{call, call::InvocResult, MarketBalance, StateManager};
+use state_tree::StateTree;
+use std::error::Error;
+
+type BoxError = Box<dyn Error + 'static>;
+pub struct MessageLookup {
+    pub receipt: MessageReceipt,
+    pub tipset: Arc<Tipset>,
+}
+pub fn get_network_name<DB>(state_manager: &StateManager<DB>) -> Result<String, BoxError>
```

Comment:
Why do you prefix others with `state_` and this one with `get_`? Seems you could get confused with the `get_network_name` from state manager